Package.swift

```diff
@@ -112,8 +117,10 @@ let permissionsTargets: [Target] = [
 let package = Package(
     name: "PermissionsSwiftUI",
     defaultLocalization: "en",
-    platforms: [.iOS(.v11)],
-    products: permissionsTargets.map{Product.library(name: $0.name, targets: [$0.name])},
+    platforms: [.iOS(.v13)],
```

Comment:
the minimum deployment of iOS 11 was for a reason, it makes it possible to integrate this package into projects that are slowly adapting SwiftUI from older UIKit. See this commit https://github.com/jevonmao/PermissionsSwiftUI/commit/b83e69f97a4b1731f50189e243cd8c8f42dcdd38